Packs/Lacework/Integrations/Lacework/Lacework.py

```diff
@@ -121,14 +129,15 @@ def get_azure_compliance_assessment():
 
     tenant_id = demisto.args().get('tenant_id')
     subscription_id = demisto.args().get('subscription_id')
+    rec_id = demisto.args().get('rec_id')
```

Comment:
```suggestion
    rec_id = argToList(demisto.args().get('rec_id'))
```

Reply:
Logic updated to leverage the argToList function.